tools/ocp_version_update/update_default_release_versions_to_latest.py

```diff
@@ -231,11 +237,13 @@ def get_release_note_url(minor_release: str):
     return release_url
 
 
-def get_release_data(minor_release):
-    release_data = subprocess.check_output(OCP_INFO_CALL.format(version=minor_release), shell=True)
+def get_release_data(minor_release, cpu_architecture):
+    if cpu_architecture == "x86_64":
+        cpu_architecture = "amd64"
```

Comment:
why we are replacing the cpu_architecture? please add a remark

Reply:
sure, added a comment